cmd/influx/pkg_test.go

```diff
@@ -202,15 +202,15 @@ func TestCmdPkg(t *testing.T) {
 					for _, labelName := range orgIDOpt.LabelNames {
 						pkg.Objects = append(pkg.Objects, pkger.Object{
 							APIVersion: pkger.APIVersion,
-							Type:       pkger.KindLabel,
+							Kind:       pkger.KindLabel,
```

Comment:
will these break any existing ones?

Reply:
should have no impact on external consumers, the `json` field is still the same, as is the pkg format. The Type part is wrong, just forgot to fix it until now